api/lib/apollo.js

```diff
@@ -85,15 +85,16 @@ async function createApolloServer(httpServer) {
             throw new Error(`Access Forbidden`)
           }
         }
-        const idToken = req.headers.authorization
-        let user = {}
-        if (idToken) {
-          user = await validateIdToken(idToken)
+
+        const sessionToken = req.headers.authorization
+        let session
+        if (sessionToken) {
+          session = await validateSession(sessionToken) // throws if session is outdated
```

Comment:
Shouldn't then we add a try/catch block in this line in `index.js`?
```
const apolloServer = await createApolloServer(httpServer)
```

Reply:
this could actually kill API altogether, throwing an error there?

Reply:
no, the error would be forwarded to the user